tests/test_tuya_builder.py

```diff
@@ -46,6 +48,41 @@ def real_device(MockAppController):
     return device
 
 
+@pytest.mark.parametrize(
+    "method_name,attr_name,exp_class",
+    [
+        ("tuya_battery", "power", TuyaPowerConfigurationCluster2AAA),
+        ("tuya_metering", "smartenergy_metering", TuyaValveWaterConsumed),
+        ("tuya_onoff", "on_off", TuyaOnOffNM),
+        ("tuya_soil_moisture", "soil_moisture", TuyaSoilMoisture),
+        ("tuya_temperature", "temperature", TuyaTemperatureMeasurement),
+        ("tuya_humidity", "humidity", TuyaRelativeHumidity),
+        ("tuya_smoke", "ias_zone", TuyaIASFire),
+        ("tuya_contact", "ias_zone", TuyaIASContact),
+    ],
+)
+async def test_convenience_methods(device_mock, method_name, attr_name, exp_class):
+    """Test TuyaQuirkBuilder convenience methods."""
+
+    registry = DeviceRegistry()
+
+    entry = TuyaQuirkBuilder(
+        device_mock.manufacturer, device_mock.model, registry=registry
+    )
+    entry = getattr(entry, method_name)(dp_id=1)
+    entry.skip_configuration().skip_configuration().add_to_registry()
```

Comment:
`skip_configuration()` is set twice right now, I guess one time is enough 😄 
```suggestion
    entry.skip_configuration().add_to_registry()
```